index.js

```diff
@@ -97,6 +153,10 @@ module.exports = postcss.plugin('postcss-responsive-type', function () {
         widthUnit = getUnit(params.minWidth),
         maxWidthUnit = getUnit(params.maxWidth);
 
+    if (sizeUnit === null) {
+      rule.warn(result, 'sizes with unitless values are not supported');
```

Comment:
We should error out here, otherwise we'll be giving them invalid CSS. No valid use-case for wanting the output.